pkg/controller/pxc/tls.go

```diff
@@ -321,9 +318,8 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLManualy(cr *api.PerconaXtraDBCl
 	data["tls.key"] = key
 	secretObjInternal := corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            cr.Spec.PXC.SSLInternalSecretName,
-			Namespace:       cr.Namespace,
-			OwnerReferences: ownerReferences,
+			Name:      cr.Spec.PXC.SSLInternalSecretName,
+			Namespace: cr.Namespace,
```

Comment:
Do you think we should cover this on an e2e test? The scenario could be that during the deletion of the cluster the SSL secret remains if given that the finalizer is disabled and the certificate autogenerated.